src/utils/constants.ts

```diff
@@ -1,5 +1,24 @@
+import { IOptions } from "@/components/Select"
+
 export const BASE_URL = 'http://localhost:8080/'
 export const DEFAULT_UNKNOWN_ERROR_MESSAGE = 'Algo deu errado!'
 export const UNKNOWN_ERROR = 'Erro desconhecido: '
 export const AXIOS_ERROR = 'Axios error: '
 export const THIRTY_DAY_COOKIE_LIFETIME = 30 * 24 * 60 * 60
+
+export enum AccountType {
```

Comment:
Enums são bacanas, porém futuramente optaria por usar Object Literals e evitar usar Enums.

Motivo 1:
Enums não são uma feature do JS, o que o typescript faz ao buildar é converter esse enum em uma IIFE, e isso 

Se quiser saber mais a fundo:

Abaixo tem vários outros motivos
https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh

Reply:
Muito bom saber disso vi no artigo , que uso de object literals é  mais escalável que o uso  de enum